src/login/loginclif.cpp

```diff
@@ -378,7 +378,7 @@ static int logclif_parse_reqkey(int fd, struct login_session_data *sd){
 	RFIFOSKIP(fd,2);
 	{
 		memset(sd->md5key, '\0', sizeof(sd->md5key));
-		sd->md5keylen = (uint16)(12 + rnd() % 4);
+		sd->md5keylen = rnd_value(12u, 15u);
```

Comment:
Is this intended to be in range [12, 15]? or was it meant to be [12, 16] but they implemented it wrong?